conda/base/context.py

```diff
@@ -206,6 +206,7 @@ class Context(Configuration):
     # ######################################################
     deps_modifier = PrimitiveParameter(DepsModifier.NOT_SET)
     update_modifier = PrimitiveParameter(UpdateModifier.UPDATE_SPECS)
+    sat_solver = PrimitiveParameter(None, element_type=string_types + (NoneType,))
```

Comment:
If we were to make this more official, I’d probably want to turn it into an enum, following the example for `channel_priority`. It’s fine for now though.